Classes/Widgets/ApcuCacheWidget.php

```diff
@@ -44,9 +44,10 @@ protected function loadData(): void
         {
             $apcuData = apcu_sma_info();
             $this->widgetEnabled = True;
-            $this->usedMemory = floatval(number_format(($apcuData["seg_size"] - $apcuData["avail_mem"])/1024/1024, self::decimals));
+            $total = $apcuData['num_seg']*$apcuData["seg_size"];
```

Comment:
Unrelated to this change, but nevertheless: `apcu_sma_info()` may return `false` on certain (not really documented) conditions. Please add a check for that condition.